core/arch/arm/plat-ls/main.c

```diff
@@ -105,6 +105,8 @@ void console_init(void)
 	pl011_init(&console_data, CONSOLE_UART_BASE, CONSOLE_UART_CLK_IN_HZ,
 		   CONSOLE_BAUDRATE);
 #else
+	console_data.io_width = IO_WIDTH_U8;
+	console_data.reg_shift = 0;
```

Comment:
Minor: personally I would not add those, since `console_data` is guaranteed to be zero-initialized here.

Reply:
As per @jenswi-linaro comment, I will rather pass them as params to `ns16550_init()`.